### ISSUE TITLE
Handling crd's in the helm chart

### ISSUE THREAD

**Issue by denniskern** [enix/kube-image-keeper#284]
Hi all

I had some trouble with the crd's and the actual helm chart respectively install it, de-install and re-install it.
The crd was removed after the re-install and also the cached image objects.

My suggestion is instead of using a variable which controls the installation of the crd's lets use the approach which
helm offers. https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you

In short: Instead of putting the crd's into the template folder put them into a separate crd folder.
Then helm take care if its needed to install the crd or not.

I would create a PR - What do you think?

**Comment by Nicolasgouze** [comment]
Hi @denniskern, 

Thanks a lot for your feedback ! 
We pinpointed this issue/behavior a couple of days back but did not have the opportunity to correct it yet ! 
Yes, you can create a PR & we’ll have a look at it ! 

Please, have a look at the small rules depicted here : 
https://github.com/enix/kube-image-keeper/blob/main/CONTRIBUTING.md

**Comment by donch** [comment]
Thanks for your contribution to the project 🙏 
I'll close the issue as it will be available in the next release.
